src/dualscreen.tsx

```diff
@@ -34,9 +34,10 @@ export class DualScreen extends KalturaPlayer.core.BasePlugin {
     this._player = player;
     this._addBindings();
     this._secondaryKalturaPlayer = this._createDummyChildPlayer();
-    this._secondaryKalturaPlayer.loadMedia({entryId: '1_q31s7nkk'});
+    this._secondaryKalturaPlayer.loadMedia({entryId: '0_4rvdnj48'});
```

Comment:
remove from the final PR as it is not part of the change and will just cause a conflict